src/lua-function-list.json

```diff
@@ -31,20 +31,24 @@
     "clearSpecialExits": "clearSpecialExits(roomID)",
     "clearUserWindow": "clearUserWindow(name)",
     "clearWindow": "clearWindow([windowName])",
+    "closeMapWidget": "closeMapWidget()",
     "closeMudlet": "closeMudlet()",
     "connectExitStub": "connectExitStub(fromID, direction) or connectExitStub(fromID, toID, [direction])",
     "connectToServer": "connectToServer(host, port, [save])",
     "copy": "copy([windowName])",
+    "copy2decho": "copy2decho([window], [stringToCopy], [instanceToCopy])",
```

Comment:
This (and some of the others) look as though the commas are NOT optional...!

Reply:
Yes, because it is easier to read this way. Has been like this for ages in wiki.

Reply:
Especially when you have something like this, where there are 3 options and -all- of them are optional.